### ISSUE TITLE
periodic reset should be possible to toggle off

### ISSUE THREAD

**Issue by jarlah** [x-actly/Eve-Online-Mining-Bot#9]
i would rather sit in a belt doing nothing with drones flying around, than trying to focus on and mine roids that are too far away. So maybe we could add an option to deactivate periodic reset of mining operation? this will also fill the hold faster (a tiny bit faster)

**Comment by jarlah** [comment]
we can also consider double clicking the roids and sleeping before targeting and mining them. This requires of course that the mining tab only contains roids, which i have filtered mine to only contain

**Comment by jarlah** [comment]
I have some time in mornings and evenings. I am playing eve a bit lately, so the interest focuses around the gaming. If I stop gaming, maybe I will have added some value to this bot for others to enjoy. For me its equally fun to just play around with the bot and the code, as it is to actually play the game ;)

**Comment by sillychipmunk** [comment]
nah, I won't do anything with this in near or far future .. its working really well now with shorter resets.
